lib/components/Radio/Radio.migration.md

```diff
@@ -0,0 +1,26 @@
+# Radio Migration Guide
+
+## API Changes
+
+- `message={false}` is now `reserveMessageSpace={false}`.
```

Comment:
We removed the field message from Radio, so this is out of date and can be removed.